### PR TITLE
Updated My Portfolio with responsive desktop version

### PR DESCRIPTION
**Pull request by V-Blaze**
I updated the My portfolio webpage and added responsive styling for desktops using media queries. Below are the major changes:

- Removed the mobile menu and added a desktop menu for bigger screen devices
- Added responsive Styling to the header section for a better desktop experience.
- Added a Different layout to the work section for the desktop version
- Made the about-me and contact section responsive to a bigger screen.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,173 +16,189 @@
...
-                <li>
-                    <a href="https://github.com/V-Blaze"><img src="./assets/images/github_icon.svg" alt="Github" a></li>
-                <li>
-                    <a href="http://"><img src="./assets/images/Vector.svg" alt="Vector"></a>
-                </li>
-            </ul>
+            <div class="headline-content">
+                <h1 class="greeting">
+                    I’m Valentine Glad to see you!
+                </h1>
```
Comment (jerryowusu) [inline index.html:29]: It looks almost all your `p` tags and `hx` tags is centred but it is supposed to be aligned to the left. Kindly go thru your project and correct this. Please note that almost all of the paragraphs (`p` tags) and headers (`hx` tags) in the design is supposed to be left aligned. Below are just a few pictorial examples:




Your design             |  Figma design
:-------------------------:|:-------------------------:
Screenshot from 2022-09-28 16-25-09  |  Screenshot from 2022-09-28 16-20-56



Your design             |  Figma design
:-------------------------:|:-------------------------:
Screenshot from 2022-09-28 16-18-11  |  Screenshot from 2022-09-28 16-21-12



Your design             |  Figma design
:-------------------------:|:-------------------------:
Screenshot from 2022-09-28 16-19-08  |  Screenshot from 2022-09-28 16-21-26

File: style.css [hunk 2]
```diff
@@ -362,3 +373,166 @@ textarea {
...
+
+@media (min-width: 768px) {
+  .mobile-menu {
+    display: none;
+  }
+
+  .nav-bar {
+    padding: 25px 155px 25px 115px;
+    align-items: center;
+  }
```
Comment (jerryowusu) [inline style.css:386]: At certain breakpoints of the media screen, the design does not look like the figma design with lots of variations and distortions whiles it looks perfectly fine at other breakpoints. You can consider adding other media query breakpoints and flexbox to make your app more responsive. Check the images below at to see some of the distortions at breakpoint 770px to 990px

Your design             |  Figma design
:-------------------------:|:-------------------------:   
Screenshot from 2022-09-28 16-49-33  |  Screenshot from 2022-09-28 16-49-57


Your design             |  Figma design
:-------------------------:|:-------------------------:
Screenshot from 2022-09-28 16-49-23  |  Screenshot from 2022-09-28 16-50-27

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @V-Blaze 

Good job so far!
Your project is looking fantastic :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

so close

## Highlights
:heavy_check_mark: Followed GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: All Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by V-Blaze** [comment]
Below are the major corrections made to the code as requested by the reviewer:

- [ ] Aligned the p tags and hx tags to the left in line with the Figma file
- [ ] Added media queries at min-width: 768px and 992px for better responsiveness on different screen sizes.

**Comment by Gopxfs** [comment]
# 🟢 STATUS: APPROVED 🟢

## Hello @V-Blaze 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**